README.md

```diff
@@ -43,9 +43,13 @@ the following text snippet to the bootloader's (e.g. GRUB) kernel command line:
 ```sh
   systemd.mask=eruption.service
 ```
-or prevent the service from starting by disabling
+Or with systemctl to mask/disable the service:
 ```sh
-$ sudo systemctl disable eruption
+$ sudo systemctl mask eruption
+```
+You can always re-enable the Eruption service with the command:
+```sh
+$ sudo systemctl enable eruption
```

Comment:
"sudo systemctl unmask eruption.service" would be the complement to "sudo systemctl mask eruption.service"